File: 1752-check-if-array-is-sorted-and-rotated/1752-check-if-array-is-sorted-and-rotated.cpp
class Solution {
public:
    bool check(vector<int>& nums) {
        int k = 0;int N = nums.size();
        for(int i = 0;i<N;i++){
            if (nums[i]>nums[(i+1)%N]) k++;
            if (k>1){
                return false;
            }
        }
        return true;
    }
};

File: 1752-check-if-array-is-sorted-and-rotated/1752-check-if-array-is-sorted-and-rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        N = len(nums);k =0
        for i in range(N):
            if nums[i]>nums[(i+1)%N]:
                k += 1
                if k> 1:
                    return False
        return True